### ISSUE TITLE
Possibility to map undefined keys (EV_KEY 300, 301)

### ISSUE THREAD

**Issue by Kugelblitz360** [KarsMulder/evsieve#26]
tl:dr - I want to grab and reroute the Key "300" that currently is undefined - but "key" can not be asked for code numbers, just the translated values.

I recently bought the Taito Trackball and Paddle controller and want to use it with Arcade emulation software on Linux. The controller has a HIGHLY unusual setup. It has a trackball that gives proper REL_X / REL_Y output and a spinner that gives REL_WHEEL. But the five buttons are partially mapped to UNUSED keycodes. The two main buttons are on 300 and 301, three smaller buttons are on 294, 295 and 297 which do have BTN equivalents. I can't find any syntax to get evsieve to capture 300 and 301 (evtest shows them alright). Any idea on how I could capture them? Or do you need to code that into the parser?

Many thanks, Boris

**Comment by Kugelblitz360** [comment]
Ah, this is closed but not in release!
